docs/related-tools/local-analysis/client-side-tools.md

```diff
@@ -0,0 +1,30 @@
+# Client-side tools
+
+Client-side tools enable you to run analysis either locally or as part of your CI process and integrating the results into your Codacy workflow. This way, Codacy presents the analysis information reported by your local tools alongside all other code quality information on the Codacy dashboards.
+
+Codacy supports client-side tools in two ways:
+
+-   **Standalone tools**
+
+    Codacy provides auxiliary converters that parse the output of third-party tools, convert the output format, and upload the results to Codacy. You must download, configure, and run the third-party tools yourself.
```

Comment:
Standalone tools just convert the output and prints it to stdout. The upload part is a manual step that should be done by the user. Maybe we should rephrase this to something like:

```suggestion
    Codacy provides auxiliary converters that parse the output of third-party tools and converts it to its format, which can then be uploaded to Codacy. You must download, configure, and run the third-party tools yourself.
```

Reply:
Thanks @josemiguelmelo, this is an important detail that also helps to clarify the difference between the two types of tools.